### PR TITLE
feat: merge renderAndGetWindow options

### LINKED ISSUE #3264: renderAndGetWindow arg #2 not work
### Version

[v1.4.0](https://github.com/nuxt/nuxt.js/releases/tag/v1.4.0)

### Reproduction link

[https://github.com/nuxt/nuxt.js/blob/dev/lib/core/renderer.js#L394](https://github.com/nuxt/nuxt.js/blob/dev/lib/core/renderer.js#L394)

### Steps to reproduce

in this code, opts hasnt used.

i think `opts` should be merged with `options`.

### What is expected ?

2nd argument `opts` should be merged with `options`.

### What is actually happening?

opts never used 

<div align="right"><sub><em>This bug report is available on <a href="https://nuxtjs.cmty.io">Nuxt.js</a> community (<a href="https://nuxtjs.cmty.io/nuxt/nuxt.js/issues/c6965">#c6965</a>)</em></sub></div>

### PR DESCRIPTION
**Pull request by TheAlexLichter**
Merging the default options with the user-defined options object gives the opportunity to use a custom virtual console and to customize jsdom-options further

Resolves #3264

### CODE REVIEW

File: lib/core/renderer.js [hunk 1]
```diff
@@ -413,17 +413,19 @@ export default class Renderer {
...
         // Mock window.scrollTo
         window.scrollTo = () => {}
       }
-    }
+    }, opts)
     const jsdomErrHandler = (err) => { throw err }
-    if (opts.virtualConsole !== false) {
-      options.virtualConsole = new jsdom.VirtualConsole().sendTo(consola)
+    if (options.virtualConsole !== true) {
+      if (options.virtualConsole === true) {
```
Comment (clarkdo) [inline lib/core/renderer.js]: Dead code?

**Review by clarkdo — CHANGES_REQUESTED** [review]

File: lib/core/renderer.js (hunk 1 above)
Comment (manniL) [reply lib/core/renderer.js]: Whoops. Should be `!== false`

Comment (manniL) [reply lib/core/renderer.js]: But then I can omit the explicit comparison. Now it should be fine

**Review by clarkdo — COMMENTED** [review]
How about adding some test for asserting options are set?

**Comment by manniL** [comment]
Will add @clarkdo

**Review by clarkdo — APPROVED** [review]

**Pull request merged**